### PR TITLE
clarify usage of mustache in readme

### PR DESCRIPTION
**Pull request by Terkwood**

### CODE REVIEW

**Comment by schowsf** [comment]
version bump needed

File: example/data/sample_workflow_view.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (realstraw) [inline example/data/sample_workflow_view.json:1]: we have the content of this file in readme, is it necessary to have it also here too?

File: README.md [hunk 2]
```diff
@@ -100,16 +98,9 @@ Below is an example of
...
-POST http://localhost:9001/v1/workflow
-```
-Which returns a workflow_id. For example: `wf-f231a08f-60e4-480a-b845-e53e06918f77`
+[sample_workflow.json.mustache](./example/data/sample_workflow.json.mustache) contains a mustache template which can accept these substitutions.
 
-Once defined, activate a workflow using the workflow id like so:
-```html
-PUT http://localhost:9001/v1/workflow/wf-f231a08f-60e4-480a-b845-e53e06918f77
-```
+Once generated, your `sample_workflow.json` can be used to create a workflow via `POST http://localhost:9001/v1/workflow`.
```
Comment (schowsf) [inline README.md:103]: Let's keep the activate part in the original README

`Once defined, activate a workflow using the workflow id like so:`
```html
PUT http://localhost:9001/v1/workflow/wf-f231a08f-60e4-480a-b845-e53e06918f77
```

Comment (Terkwood) [reply README.md]: Moved the text around a bit to accommodate your suggestion

**Review by schowsf — APPROVED** [review]
LGTM

**Review by realstraw — APPROVED** [review]

**Pull request merged**